xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -546,6 +562,7 @@ extern "C" {
     bool   bIsDeleted;                                    /*!< @brief (optional) shows this recording is deleted and can be undelete */
     unsigned int iEpgEventId;                             /*!< @brief (optional) EPG event id associated with this recording. Valid ids must be greater than EPG_TAG_INVALID_UID. */
     int    iChannelUid;                                   /*!< @brief (optional) unique identifier of the channel for this recording. PVR_CHANNEL_INVALID_UID denotes that channel uid is not available. */
+    int    iRecordingState;                               /*!< @brief (optional) the recording state of the recorded TV item. PVR_RECORDED_TV_ITEM_STATE_INVALID denotes that the recording state is not available */
```

Comment:
If you introduce an enum for the state, use it! ;-)

And, even more important, this must a required property.